### PR TITLE
Added passlib.travis.yml ,mothur.trayis.yml

### PR DESCRIPTION
**Pull request by ujjwalsh**
Hi ,
I have Added travis.yml for passlib repository.

### CODE REVIEW

**Comment by gururajrkatti** [comment]
Can you pl. squash the commits.

**Comment by ujjwalsh** [comment]
> Can you pl. squash the commits.

Hi Guru ,
I have squashed the commits .
Regards,
Ujjwal

File: travis-ymls/mothur.travis.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : mothur
+# Source Repo         : https://github.com/ujjwalsh/mothur
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/mothur.travis.yml:4]: Please add main repo link and not your fork.

File: travis-ymls/mothur.travis.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Travis Job Link     : https://travis-ci.com/github/ujjwalsh/mothur/builds/213262386
+# Created travis.yml  : N
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/mothur.travis.yml:6]: Typo

File: travis-ymls/mothur.travis.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Maintainer          : Ujjwal Sharma
+#
+# Script License      : g++
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/mothur.travis.yml:9]: This should be always `Apache License, Version 2 or later`.

File: travis-ymls/passlib.travis.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : passlib
+# Source Repo         : https://github.com/ujjwalsh/passlib
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/passlib.travis.yml:4]: Same as above.

File: travis-ymls/passlib.travis.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+# Travis Job Link     : https://travis-ci.com/github/ujjwalsh/passlib/builds/213255787 
+# Created travis.yml  : N
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/passlib.travis.yml:6]: Same as above.

File: travis-ymls/passlib.travis.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+# Maintainer          : Ujjwal Sharma
+#
+# Script License      : go
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/passlib.travis.yml:9]: Same as above.

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

File: travis-ymls/mothur.travis.yml (hunk 1 above)
Comment (ujjwalsh) [reply travis-ymls/mothur.travis.yml]: Done

File: travis-ymls/mothur.travis.yml (hunk 2 above)
Comment (ujjwalsh) [reply travis-ymls/mothur.travis.yml]: done

File: travis-ymls/mothur.travis.yml (hunk 3 above)
Comment (ujjwalsh) [reply travis-ymls/mothur.travis.yml]: done

File: travis-ymls/passlib.travis.yml (hunk 6 above)
Comment (ujjwalsh) [reply travis-ymls/passlib.travis.yml]: done

File: travis-ymls/passlib.travis.yml (hunk 5 above)
Comment (ujjwalsh) [reply travis-ymls/passlib.travis.yml]: done

File: travis-ymls/passlib.travis.yml (hunk 4 above)
Comment (ujjwalsh) [reply travis-ymls/passlib.travis.yml]: done

**Comment by ujjwalsh** [comment]
Hi @Siddhesh-Ghadi  @gururajrkatti All the requested changes has been done. Please review & merge 
Thanks.

**Comment by Siddhesh-Ghadi** [comment]
Ready for merge @seth-priya

**Pull request merged**